source/mongodump/mongodump-behavior.txt

```diff
@@ -32,6 +32,8 @@ mongodump Behavior, Access, and Usage
    the :dbtools:`--archive </mongodump/#std-option-mongodump.--archive>` 
    option.  
 
+.. include:: /includes/warning-tools-dot-dollar-import-export-restriction.rst
```

Comment:
Although similar, I don't think it's the exact same warning that we want to show on the mongodump and mongorestore pages. Mongodump and mongrestore are not affected if document field names contain dots or if `$`-prefixed document field names are not in a collection's option. It's only `$`-prefixed document field names in a **collection's option** that users shouldn't have when using mongodump and mongorestore.

Reply:
Adjusted.